api_volontaria/apps/volunteer/models.py

```diff
@@ -184,6 +226,11 @@ class Meta:
         on_delete=models.PROTECT,
     )
 
+    tags = models.ManyToManyField(
+        Tag,
+        related_name='tags',
```

Comment:
The `related_name` is the reverse naming, so it will be use like `Tag.your_choice`. I think it should be `events` here

Reply:
@RignonNoel I'm facing a problem with this : Test that we can add existing tag during event creation (and that it does not create a duplicated tag)

I have created and commented a test line 144 of test_view_events.py.

Using WritableNestedModelSerializer seem to create the problem that I cannot add in a m2m an array of link to existing tags.  

I have tried several approches to POST a new event with existing TAG without result.  It's always asking to enter the name of the tag.  

A strategy would be appreciated !